tests/test_word_tokenize.py

```diff
@@ -48,58 +49,64 @@ def test_word_tokenize_with_sentencepiece(self):
-        self.assertEqual(expect, result)
-
-
-class WordTokenizerWithLowerCaseTest(unittest.TestCase):
-    """Test ordinal word tokenizer."""
-
-    def test_word_tokenize_with_kytea(self):
-        """Test KyTea tokenizer."""
+    def test_word_tokenize_with_sudachi_mode_a(self):
+        """Test Sudachi tokenizer."""
         try:
-            tokenizer = WordTokenizer(tokenizer="kytea")
+            tokenizer = WordTokenizer(
+                tokenizer="Sudachi",
+                mode="A",
+            )
         except ModuleNotFoundError:
-            pytest.skip("skip kytea")
+            pytest.skip("skip sudachi")
 
-        expect = [Token(surface=w) for w in "吾輩 は 猫 で あ る".split(" ")]
-        result = tokenizer.tokenize(SENTENCE1)
+        expect = [Token(surface=w) for w in "医薬 品 安全 管理 責任 者".split(" ")]
+        result = tokenizer.tokenize(SENTENCE2)
         self.assertEqual(expect, result)
 
-    def test_word_tokenize_with_mecab(self):
-        """Test MeCab tokenizer."""
+    def test_word_tokenize_with_sudachi_mode_b(self):
+        """Test Sudachi tokenizer."""
         try:
-            tokenizer = WordTokenizer(tokenizer="mecab")
+            tokenizer = WordTokenizer(
+                tokenizer="Sudachi",
+                mode="B",
+            )
         except ModuleNotFoundError:
-            pytest.skip("skip mecab")
+            pytest.skip("skip sudachi")
 
-        expect = [Token(surface=w) for w in "吾輩 は 猫 で ある".split(" ")]
-        result = tokenizer.tokenize(SENTENCE1)
+        expect = [Token(surface=w) for w in "医薬品 安全 管理 責任者".split(" ")]
+        result = tokenizer.tokenize(SENTENCE2)
         self.assertEqual(expect, result)
 
-    def test_word_tokenize_with_sentencepiece(self):
-        """Test Sentencepiece tokenizer."""
+    def test_word_tokenize_with_sudachi_mode_c(self):
+        """Test Sudachi tokenizer."""
         try:
             tokenizer = WordTokenizer(
-                tokenizer="Sentencepiece",
-                model_path="data/model.spm"
+                tokenizer="Sudachi",
+                mode="C",
             )
         except ModuleNotFoundError:
-            pytest.skip("skip sentencepiece")
+            pytest.skip("skip sudachi")
 
-        expect = [Token(surface=w) for w in "▁ 吾 輩 は 猫 である".split(" ")]
-        result = tokenizer.tokenize(SENTENCE1)
+        expect = [Token(surface=w) for w in "医薬品安全管理責任者".split(" ")]
+        result = tokenizer.tokenize(SENTENCE2)
         self.assertEqual(expect, result)
 
     def test_word_tokenize_with_character(self):
         """Test Character tokenizer."""
-        tokenizer = WordTokenizer(tokenizer="character")
+        tokenizer = WordTokenizer(
+            tokenizer="Character"
+        )
         expect = [Token(surface=w) for w in "吾 輩 は 猫 で あ る".split(" ")]
         result = tokenizer.tokenize(SENTENCE1)
         self.assertEqual(expect, result)
+
+    def test_word_tokenize_using_lowercase(self):
```

Comment:
Testing for one case is enough for specifying a lower cased name of a tokenizer.